app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/dialogs/SendGeographicMessageDialog.java

```diff
@@ -71,6 +85,7 @@ public void onCreate(Bundle savedInstanceState) {
 
         if (arguments != null) {
             mPoint = arguments.getParcelable(ARG_POINT_GEOMETRY);
+            mText = arguments.getString(ARG_POINT_TEXT);
```

Comment:
Nowhere in the code the text is saved into the fragment's argument.
This line is redundant, and so does the static String ARG_POINT_TEXT.
!fix